jina/docker/hubio.py

```diff
@@ -51,7 +51,7 @@ def __init__(self, args: 'argparse.Namespace'):
             self.logger.critical('requires "docker" dependency, please install it via "pip install jina[docker]"')
             raise
 
-    def new(self):
+    def _get_cookiecutter_template(self):
```

Comment:
is there any option to add tests for these functionalities under `tests/integration`?

Reply:
At the moment, there are no tests for hub sub commands in `tests/integration`, there is `https://github.com/jina-ai/jina/blob/master/tests/unit/test_cli.py`, however, this also doesn't test the commands. I could open an issue for tests for all `jina hub [subcommands]`